lib/addExpensePage.dart

```diff
@@ -33,7 +33,18 @@ class _AddExpensePageState extends State<AddExpensePage> {
               TextFormField(
                 decoration: InputDecoration(labelText: 'Name:'),
                 onSaved: (input) => _name = input,
-                validator: (input) => input.isEmpty ? 'enter value' : null,
+
+                validator: (input) {
+                  if(input.length == 0){return 'Adaugati Valoare';}
+                  else{
+                  var check = true;
+                  for(int i=0;i<input.length;i++){
```

Comment:
please use input formatters if you want to restricts or allow some characters instead of this.

ex:
```
TextField(inputFormatters: [
    WhitelistingTextInputFormatter(RegExp("[a-zA-Z]")),
],)
```